products-1.html

```diff
@@ -11,17 +11,17 @@
 			<nav name="top">
 				<ul>
 					<li><a href="quiz.html">skin type quiz</a></li>
-					<li class="dropdown"><a href="all-products.html" class="current">products &#9662;</a>
+					<li class="dropdown"><a href="products-1.html" class="current">products &#9662;</a>

```

Comment:
I changed the name to "products-1" because it doesn't contain all the products like my old file name implied, so this new one is more relevant.